resources/organizations_accounts.go

```diff
@@ -83,7 +83,11 @@ func fetchOrganizationsAccounts(ctx context.Context, meta schema.ClientMeta, par
 		if errors.As(err, &ae) {
 			switch ae.ErrorCode() {
 			case "AWSOrganizationsNotInUseException", "AccessDeniedException":
-				return diag.FromError(err, diag.IGNORE, diag.ACCESS, OrganizationsAccounts().Name, client.ParseSummaryMessage(c.Accounts, err, ae), "Missing permissions or account might not be root/organizational unit.")
+				// This is going to happen most proabbly due to account not being the root organizational account
+				// so it's better to ignore it completly as it happens basically on every account
+				// otherwise it screws up with dev experiance and with our tests
+				log.Warn("account is most probabbly not the root organization account https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListAccounts.html")
```

Comment:
```suggestion
				log.Warn("account is most probabbly not the root organization account https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListAccounts.html")
```
```suggestion
				log.Warn("account is most probably not the root organization account https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListAccounts.html")
```

(with the typo fixed the wording still isn't the best)